python-sdk/src/astro/sql/operators/export_file.py

```diff
@@ -62,6 +73,77 @@ def execute(self, context: Context) -> File:
         else:
             raise FileExistsError(f"{self.output_file.path} file already exists.")
 
+    def get_openlineage_facets(self, task_instance) -> OpenLineageFacets:  # skipcq: PYL-W0613
+        """
+        Returns the lineage data
```

Comment:
Adding some more insight would be helpful

Reply:
Added more info